R/pr.R

```diff
@@ -376,13 +385,14 @@ pr_push <- function() {
     # this is the first push
     if (cfg$type == "fork" && cfg$upstream$can_push && is_interactive()) {
       choices <- c(
-        origin   = glue(
-          "{cfg$origin$repo_spec} = {ui_value('origin')} (external PR)"),
-        upstream = glue(
-          "{cfg$upstream$repo_spec} = {ui_value('upstream')} (internal PR)")
+        origin   = ui_pre_glue("
+          <<cfg$origin$repo_spec>> = {.val origin} (external PR)"),
+        upstream = ui_pre_glue("
+          <<cfg$upstream$repo_spec>> = {.val upstream} (internal PR)")
       )
+      choices_formatted <- map_chr(choices, cli::format_inline)
       title <- glue("Which repo do you want to push to?")
-      choice <- utils::menu(choices, graphics = FALSE, title = title)
+      choice <- utils::menu(choices_formatted, graphics = FALSE, title = title)
       remote <- names(choices)[[choice]]
```

Comment:
Prepping the inputs for `utils::menu()` is sort of painful. Lots of pre-interpolating data, then `cli::format_inline()` to get styling, so we have everything ready before handing over to `utils::menu()`. usethis would certainly make good use of a cli function that helps the user choose between things.